server/main.lua

```diff
@@ -44,9 +44,20 @@ end)
 lib.addCommand('startshow', {
     help = locale('start_show_desc'),
     params = {
-        {name = "Location", help = locale('start_show_param_hint'), type = 'string'}
+        { name = "Location", help = locale('start_show_param_hint'), type = 'string' }
     },
     restricted = 'group.admin'
 }, function(source, args, raw)
-    TriggerClientEvent('qbx_fireworks:client:startShow', source, args.Location)
+    local show = args.Location
+    if not show then return end
+    if not config.shows[show] then return end
+    for i = 1, #config.shows[show].fireworks do -- Sequences
```

Comment:
Consider extracting config.shows[show] to a variable.

Reply:
outdated